src/itdelatrisu/opsu/objects/HitResultType.java

```diff
@@ -0,0 +1,10 @@
+package itdelatrisu.opsu.objects;
+
+public enum HitResultType {
+    CIRCLE,
+    SLIDERSTART,
```

Comment:
That was going to be used, but I never actually used it. I left it in there in the event that anyone in the future wanted to use it.